--- php-cli.ini.old	2019-01-27 08:35:16.818588355 -0800
+++ php-cli.ini.new	2019-02-04 20:23:43.012688055 -0800
@@ -1359,7 +1359,7 @@
 ; Memcached
 ;session.save_path = "localhost:11211"
 ; Redis
-session.save_path = "tcp://127.0.0.1:6379"
+session.save_path = "tcp://redis:6379"
 
 ; Whether to use strict session mode.
 ; Strict session mode does not accept uninitialized session ID and regenerate
